ChangeLog.md

```diff
@@ -1,6 +1,15 @@
 # DGtal 0.9
 
 ## New Features / Critical Changes
+- *Geometry Package*
+
+ - New Alpha Thick Segment Computer allowing the recognition of thick digital segment,
```

Comment:
New segment computer allowing the recognition of thick digital segments, adapted to noisy contours (from a given thickness parameter). The current implementation (mainly a backport from  imagene) is a model of CForwardSegmentComputer with a ParallelStrip primitive. This primitive is similar to the blurred segment of [Debled-Rennesson etal 2005] with isothetic thickness. (Bertrand Kerautret, [#963](https://github.com/DGtal-team/DGtal/pull/963))